specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json

```diff
@@ -1763,6 +1805,133 @@
         "ids"
       ],
       "description": "Parameters for a Redis Enterprise Active Geo Replication Force Unlink operation."
+    },
+    "RegionSkuDetail": {
+      "title": "Region Sku Detail",
+      "description": "Details about the location requested and the available skus in the location",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string"
+        },
+        "locationInfo": {
+          "$ref": "#/definitions/LocationInfo",
+          "description": "Details about location and its capabilities"
+        },
+        "skuDetails": {
+          "$ref": "#/definitions/SkuDetail",
+          "description": "Details about available skus"
+        }
+      }
+    },
+    "RegionSkuDetails": {
+      "title": "Region Sku Details",
+      "description": "List of Region SKU Detail",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegionSkuDetail"
+          },
+          "description": "List of SkuDetails"
+        }
+      }
+    },
+    "LocationInfo": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string"
+        },
+        "zoneMappings": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ZoneMapping"
+          }
+        },
+        "locationCapabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          }
+        }
+      }
+    },
+    "SkuDetail": {
+      "type": "object",
+      "title": "Sku Detail",
+      "description": "Information about Sku",
+      "properties": {
+        "skuName": {
+          "type": "integer",
+          "format": "int64"
+        },
+        "name": {
+          "type": "string",
+          "description": "The type of RedisEnterprise cluster to deploy. Possible values: (Enterprise_E10, EnterpriseFlash_F300 etc.)",
+          "enum": [
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500"
+          ],
+          "x-ms-enum": {
+            "name": "Name",
+            "modelAsString": true
+          }
+        },
+        "defaultMaxMemory": {
+          "type": "number"
+        },
+        "defaultMaxFlash": {
+          "type": "number"
+        },
+        "planId": {
+          "type": "string"
+        },
+        "internalPlanId": {
+          "type": "string"
+        },
+        "isFlash": {
+          "type": "boolean"
+        },
+        "isLegacySku": {
+          "type": "boolean"
+        },
+        "requiresQuorumNode": {
```

Comment:
This information is fetched from SkuDetails class. However, should all the fields be exposed?

Reply:
I have the same doubt. What are the reasons for exposing information such as maxMemory or planId in the API? We should only expose information if there's an obvious value in doing so. Definitely don't expose 'IsLegacySku', 'IsFlash', or 'RequiresQuorumNode' or 'InternalPlanId'.